proto/streamlit/proto/Heading.proto

```diff
@@ -22,4 +22,5 @@ message Heading {
 
     string anchor = 2;
     string body = 3;
+    bool hide_anchor = 4;
```

Comment:
NIT: I don't mind having more Proto at all, I guess it would be cleaner if we could just do when anchor is empty then we don't show it

Reply:
an empty anchor field has its existing meaning, so we can't re-use it.

Reply:
Ok 👍